region/src/main/java/org/motechproject/nms/region/domain/HealthFacility.java

```diff
@@ -44,8 +43,6 @@
     private Long code;
 
     @Field
-    @Column(allowsNull = "false")
-    @NotNull
     private HealthFacilityType healthFacilityType;
```

Comment:
HealthFacilityType won't come in the mcts files right?  If so are we ever going to set it?  I think it might be needed in reports